esi_leap/objects/offer.py

```diff
@@ -58,6 +62,44 @@ def get_all_by_status(cls, context, status):
             context, status)
         return cls._from_db_object_list(context, db_offers)
 
+    @classmethod
+    def send(cls, context, offer_uuid):
+        db_offer = cls.dbapi.offer_get(context, offer_uuid)
+        o = cls._from_db_object(context, cls(), db_offer)
+
+        auth = v3.Password(auth_url='http://localhost:5000/v3',
+                           username='flocx-provider',
+                           password='password',
+                           project_name='service',
+                           user_domain_id='default',
+                           project_domain_id='default')
```

Comment:
I'm guessing you already know that these will eventually have to be updated to be oslo configs : )

Reply:
Sure!

Reply:
Hi Mainn, I'm confused here: should I pass the marketplace or provider's username and password here in order to make a session?

Reply:
In the long term the Marketplace may not be the same OpenStack instance as the provider, and may provide the provider with credentials to talk to the marketplace. So what we want is for the provider to specify whatever username/password/endpoint that the marketplace says to use.